### PR TITLE
[DOCS] Update permissions needed for Endpoint Management

### LINKED ISSUE #1638: [DOCS] Update permissions needed for Endpoint Management
## Description 

Check whether we need to add/update documentation on permissions needed for "Endpoint Management" (presumably the Endpoints page, but check if other pages are involved).

Per https://github.com/elastic/kibana/pull/124118:

> The Endpoint Management tab in the Security Solution now checks for `superuser` role directly. This is because Fleet is introducing their own, finer grained Kibana permission checks. We now need to check for `superuser` directly while we define our own Kibana permissions.

## Screenshots

Old text:
image

New text:
image

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #1638. Adds a note to endpoint management pages, which require the `superuser` role to access.

Previews:
* [Endpoints](https://security-docs_1650.docs-preview.app.elstc.co/guide/en/security/master/admin-page-ov.html)
* [Host isolation](https://security-docs_1650.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-ov.html)
* [Trusted applications](https://security-docs_1650.docs-preview.app.elstc.co/guide/en/security/master/trusted-apps-ov.html)
* [Event filters](https://security-docs_1650.docs-preview.app.elstc.co/guide/en/security/master/event-filters.html)
* [Host isolation exceptions](https://security-docs_1650.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-exceptions.html)

### CODE REVIEW

**Review by kevinlog — APPROVED** [review]
thanks for adding this in!

**Review by caitlinbetz — APPROVED** [review]

**Review by nastasha-solomon — APPROVED** [review]

File: docs/management/admin/admin-pg-ov.asciidoc [hunk 1]
```diff
@@ -3,7 +3,12 @@
 = Endpoints
 The Endpoints page allows admins to view and manage endpoints that are running the <<install-endpoint, Endpoint Security integration>>.
 
-NOTE: {fleet} must be enabled in a {kib} space for administrative actions to function correctly.
+[NOTE]
+=====
+{fleet} must be enabled in a {kib} space for administrative actions to function correctly.
+
+You must have the built-in `superuser` role to access this feature. For more information, see {ref}/built-in-users.html[Built-in users].
```
Comment (jmikell821) [inline docs/management/admin/admin-pg-ov.asciidoc:10]:
```suggestion
You must have the built-in `superuser` role to access this feature. For more information, refer to {ref}/built-in-users.html[Built-in users].
```

File: docs/management/admin/event-filters.asciidoc [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 Event filters allow admins to filter endpoint events that you do not need or want stored in {es} -- for example, those with high volumes. By creating event filters, you can optimize your storage in {es}. All endpoint events have the `endpoint.events.network` field.
 
+NOTE: You must have the built-in `superuser` role to access this feature. For more information, see {ref}/built-in-users.html[Built-in users].
```
Comment (jmikell821) [inline docs/management/admin/event-filters.asciidoc:7]:
```suggestion
NOTE: You must have the built-in `superuser` role to access this feature. For more information, refer to {ref}/built-in-users.html[Built-in users].
```

File: docs/management/admin/host-isolation-exceptions.asciidoc [hunk 3]
```diff
@@ -6,6 +6,8 @@ You can configure host isolation exceptions for specific IP addresses that <<hos
 
 Host isolation exceptions support IPv4 addresses, with optional classless inter-domain routing (CIDR) notation.
 
+NOTE: You must have the built-in `superuser` role to access this feature. For more information, see {ref}/built-in-users.html[Built-in users].
```
Comment (jmikell821) [inline docs/management/admin/host-isolation-exceptions.asciidoc:9]:
```suggestion
NOTE: You must have the built-in `superuser` role to access this feature. For more information, refer to {ref}/built-in-users.html[Built-in users].
```

File: docs/management/admin/host-isolation-ov.asciidoc [hunk 4]
```diff
@@ -15,7 +15,7 @@ For {stack} version >= 7.15.0, host isolation is supported for endpoints running
 * Ubuntu 20.04
 * AWS Linux 2
 
-To isolate and release hosts in any operating system, you must have the built-in superuser role. For more information, see {ref}/built-in-users.html[Built-in users].
+To isolate and release hosts in any operating system, you must have the built-in `superuser` role. For more information, see {ref}/built-in-users.html[Built-in users].
```
Comment (jmikell821) [inline docs/management/admin/host-isolation-ov.asciidoc:18]:
```suggestion
To isolate and release hosts in any operating system, you must have the built-in `superuser` role. For more information, refer to {ref}/built-in-users.html[Built-in users].
```

File: docs/management/admin/trusted-apps.asciidoc [hunk 5]
```diff
@@ -2,7 +2,9 @@
 [chapter, role="xpack"]
 = Trusted applications
 
-Users with the `superuser` role can add Windows, macOS, and Linux applications that should be trusted. By adding these trusted applications, you can use {elastic-sec} without compatibility or performance issues with other installed applications on your system. Trusted applications are applied only to hosts running {endpoint-sec}.
+You can add Windows, macOS, and Linux applications that should be trusted. By adding these trusted applications, you can use {elastic-sec} without compatibility or performance issues with other installed applications on your system. Trusted applications are applied only to hosts running {endpoint-sec}.
+
+NOTE: You must have the built-in `superuser` role to access this feature. For more information, see {ref}/built-in-users.html[Built-in users].
```
Comment (jmikell821) [inline docs/management/admin/trusted-apps.asciidoc:7]:
```suggestion
NOTE: You must have the built-in `superuser` role to access this feature. For more information, refer to {ref}/built-in-users.html[Built-in users].
```

**Review by jmikell821 — COMMENTED** [review]
I had the same comment throughout. Not writing `see` is tough and will be hard for us to get used to :) Looks good otherwise.

**Review by benironside — APPROVED** [review]
LGTM

**Comment by joepeeples** [comment]
> I had the same comment throughout. Not writing `see` is tough and will be hard for us to get used to :) Looks good otherwise.

Great catch @jmikell821, thank you!

**Pull request merged**